zap/src/output/tooling.rs

```diff
@@ -1,7 +1,7 @@
 use std::{cmp::max, collections::HashMap};
 
```

Comment:
Can we add an irgen test so that the issue was caught? As your test did not pickup the required changes to tooling